.taskmaster/error-handler.py

```
@@ -0,0 +1,350 @@
+                        lines = result.stdout.strip().split('\n') if result.stdout.strip() else []
+                        conflicts = [line for line in lines if line.startswith('UU ') or line.startswith('AA ')]
+                        
+                        if conflicts:
+                            self.log_error(
+                                ErrorType.FILE_CONFLICT,
+                                ErrorSeverity.MEDIUM,
+                                f"Git conflicts detected in {agent}",
+                                agent_id=agent,
+                                details={"conflicts": conflicts}
+                            )
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)